airbyte-db/src/test/java/io/airbyte/db/jdbc/TestJdbcUtils.java

```diff
@@ -278,9 +278,9 @@ private static void assertExpectedOutputTypes(Connection connection) throws SQLE
     final Map<String, JsonSchemaPrimitive> expected = ImmutableMap.<String, JsonSchemaPrimitive>builder()
         .put("bit", JsonSchemaPrimitive.BOOLEAN)
         .put("boolean", JsonSchemaPrimitive.BOOLEAN)
-        .put("smallint", JsonSchemaPrimitive.NUMBER)
-        .put("int", JsonSchemaPrimitive.NUMBER)
-        .put("bigint", JsonSchemaPrimitive.NUMBER)
+        .put("smallint", JsonSchemaPrimitive.INTEGER)
+        .put("int", JsonSchemaPrimitive.INTEGER)
+        .put("bigint", JsonSchemaPrimitive.INTEGER)
```

Comment:
Does this mean we expect destinations to handle all `INTEGER` as longs?

Reply:
1. Do we need to add integer in this PR?
2. How good to we feel that everything still works after adding a new type? Just running the tests is insufficient in this case because none of the tests are testing for integer type. So it seems like we need to add tests for sources, destinations, etc to make sure that this gets supported sanely. We can do this, but that's a bigger PR (which goes back to question 1). 

My feeling is as is, this is yoloing a kinda big change. So we should know why it is valuable right now and understand why we are yoloing it versus doing it more carefully with adequate testing.

Reply:
normalization is converting a `JsonSchemaPrimitive.INTEGER` into `dbt_utils.type_bigint` (the largest integer available as we don't have info on the size of the integer anymore)

```
{% macro default__type_bigint() %}
    bigint
{% endmacro %}

{% macro bigquery__type_bigint() %}
    int64
{% endmacro %}
```
from https://github.com/fishtown-analytics/dbt-utils/blob/ceb28497769c642cae7e3d5d18f1fe6bb253ef59/macros/cross_db_utils/datatypes.sql#L71